pkg/util/mount/mount_linux.go

```diff
@@ -367,7 +367,11 @@ func (mounter *SafeFormatAndMount) formatAndMount(source string, target string,
 			}
 
 			if fstype == "ext4" || fstype == "ext3" {
-				args = []string{"-F", source}
+				if len(mountArgs) == 0 {
+					args = []string{"-F", source}
+				} else {
+					args = append(mountArgs, source)
```

Comment:
@brendandburns can you give examples of mount args? I was thinking of mount options, but you may want to pass something else.